src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,121 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private Long id;
+  private String genre;
+
+  @Column(nullable=false)
+  private  String author;
+
+  @Column(nullable=false)
+  private  String image;
+
+  @Column(nullable=false)
+  private  String title;
+
+  @Column(nullable=false)
+  private String subtitle;
+
+  @Column(nullable=false)
+  private String publisher;
+
+  @Column(nullable=false)
+  private String year;
+
+  @Column(nullable=false)
+  private  Integer pages;
+
+  @Column(nullable=false)
```

Comment:
To unique values ​​such as isbn add the property "unique = true" as follows:

@Column (nullable = false, unique = true)